yucca/training/configuration/split_data.py

```diff
@@ -40,6 +41,9 @@ def get_split_config(
         - If `k` is provided we will split with `k-fold`.
         - If `p` is provided it determines the fraction of items used for the val split.
     """
+    if stage == "predict":
```

Comment:
I would much rather have that we just change the manager such that it branches in the split_config than make a [nop](https://en.wikipedia.org/wiki/NOP_(code)) based on an argument value. From the caller side i find this very hard to understand this code. I would do this instead in `YuccaManager`:

```python
splits_config = get_split_config(train_data_dir=path_config.train_data_dir, task=task_config.task) if stage != 'predict' else SplitConfig()
```